api/namex/models/payment.py

```diff
@@ -2,14 +2,15 @@
 from enum import Enum
 from sqlalchemy import event
 
+from namex.constants import PaymentState
 from namex.models import State, db
 
 
 class Payment(db.Model):
 
     class PaymentActions(Enum):
         """Valid actions for a payment."""
-        COMPLETE='COMPLETE'

```

Comment:
this state is used by /services/namex-pay
i think CREATE was added, but maybe missed from this enum

Reply:
Yes I've included the namex-pay changes in this PR

Reply:
bah, missed the file path change. sweet.